Lang TW build failed starting from 5d9e2afcd

### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

Build failed

### Expected Behavior

Build success

### Steps To Reproduce

Build with translation TW.

### Version

Nightly (Please give date/commit below)

### Transmitter

Flysky NV14

### Anything else?

/edgetx/radio/src/storage/modelslist.cpp: In member function 'bool ModelsList::loadYaml()':
/edgetx/radio/src/storage/modelslist.cpp:1102:21: error: 'TR_MODELS_MOVED' was not declared in this scope; did you mean 'STR_MODELS_MOVED'?
 1102 |       POPUP_WARNING(TR_MODELS_MOVED "\n" UNUSED_MODELS_PATH, "\n" TR_PRESS_ANY_KEY_TO_SKIP);
      |                     ^~~~~~~~~~~~~~~
      |                     STR_MODELS_MOVED
radio/src/CMakeFiles/firmware.dir/build.make:709: recipe for target 'radio/src/CMakeFiles/firmware.dir/storage/modelslist.cpp.obj' failed
make[3]: *** [radio/src/CMakeFiles/firmware.dir/storage/modelslist.cpp.obj] Error 1


_No response_

--- a/radio/src/translations/tw.h
+++ b/radio/src/translations/tw.h
@@ -633,6 +633,7 @@
 #define TR_SELECT_MODE                 "選擇模式"
 #define TR_CREATE_MODEL                "創建模型"
 #define TR_FAVORITE_LABEL              "Favorites"
+#define TR_MODELS_MOVED                "Unused models moved to"
 #define TR_NEW_MODEL                   "New Model"
 #define TR_INVALID_MODEL               "Invalid Model"
 #define TR_EDIT_LABELS                 "Edit Labels"

